Annotation/constituent_golden_set.json

```diff
@@ -708,7 +824,7 @@
                     }
                 ],
                 "type": "constituent",
-                "clause": "",
+                "clause": "how he much he has enjoyed working for the council and nothing ever made him want to leave",

```

Comment:
This is another degree clause (and/or free relative?), so possibly not what we're looking for. At any rate, I think the "and nothing ever made him want to leave" is not part of the clause but rather another complement (he says x and y), kind of informally quoting

Reply:
I'll add the coordination, will leave for the degree clause for now